VirtoCommerce.Storefront.Model/Security/Contracts/UpdateRoleResponseDto.cs

```diff
@@ -0,0 +1,7 @@
+namespace VirtoCommerce.Storefront.Model.Security.Contracts
+{
+    public class UpdateRoleResponseDto
```

Comment:
no need for dedicated ResponseDto class in here. Create and use more general SecurityResponseDto

Reply:
look at down